packages/wallets/src/features/cfd/CFDPlatformsListEmptyState.tsx

```diff
@@ -20,7 +20,15 @@ const CFDPlatformsListEmptyState = () => {
                 To trade CFDs, you’ll need to use your {fiatAccount?.wallet_currency_type} Wallet. Click Transfer to
                 move your {activeWallet?.currency} to your {fiatAccount?.wallet_currency_type} Wallet.
             </WalletText>
-            <WalletButton color='primary-light' onClick={() => history.push('/wallets/cashier/transfer')} size='lg'>
+            <WalletButton
+                color='primary-light'
+                onClick={() =>
+                    history.push('/wallets/cashier/transfer', {
+                        shouldSelectDefaultWallet: true,
```

Comment:
Only when we click `Transfer` from an empty state in a crypto wallet, we only select default wallet to USD wallet

https://github.com/binary-com/deriv-app/assets/103016120/7a3192c3-348c-4870-9f39-312dee7f8a8c